apps/condo/utils/clientSchema/Ticket/Status.ts

```diff
@@ -0,0 +1,26 @@
+import { genReactHooks } from '@core/keystone/gen.gql.react.utils'
+import { TicketStatus } from '../../../schema/Ticket.gql'
+
+function convertGQLItemToUIState (item) {
+    if (item.dv !== 1) throw new Error('unsupported item.dv')
+    return { ...item, dv: undefined }
+}
+
+function convertUIStateToGQLItem (item) {
+    return item
+}
+
+function convertGQLItemToFormSelectState (item) {
```

Comment:
@pahaz is it ok to place function's like this here in `clientScheema` utils, or should we delegate this work to single `convertGQLItemToUIState()` ?

Reply:
I think it's ok. I'm a little confused by the naming. `item` is the result of `convertGQLItemToUIState` it's `UIState`. maybe it's better to name it like `contertToFormSelectState` or `convertUIStateToFormSelectState` or etc...

Reply:
There is one more important point. What we need to do if we needed to define a `convertGQLItemToFormSelectState` function for different models. For example for `TicketStatus` and `TicketSource`. Maybe it's better to export it as: 
```
const TicketStatus = genReactHooks(TicketStatus, { convertGQLItemToUIState, convertUIStateToGQLItem })
TicketStatus.convertGQLItemToFormSelectState = ...
module.exports = {
    TicketStatus,
}
```